### ISSUE TITLE
Update qubes-hcl-report and web page table for 4.x requirements

### ISSUE THREAD

**Issue by andrewdavidwong** [QubesOS/qubes-issues#3214]
Twitter user [@gnsthegrn](https://twitter.com/gnsthegrn) has [suggested](https://twitter.com/gnsthegrn/status/923286166782300160) that we update the HCL web page table for the new 4.x requirements and provided [this example Google spreadsheet](https://docs.google.com/spreadsheets/d/1h8Ko-NIhh9r3h52yatt85UFDzBWj3kkk2MI6zmkAQO8/).

This would require that we update the `qubes-hcl-report` tool in order to gather the new data. It already checks for SLAT now, but perhaps other things are missing.

What do you think, @marmarek and @zrubi?

-----

Potentially related issues: #2214,

**Comment by tasket** [comment]
FWIW, web page already shows SLAT.

Remap is collected, but not yet shown. I'm guessing this one is important...

Non-USB keyboard isn't strictly necessary to use R4, but I know there is an Issue somewhere to collect that data.

'USB connected mic' is intriguing, but is this necessary?

Overall, this + other suggested additions may be too much detail to fit in the main table. But perhaps some kind of tooltip or pop-over could be used.

**Comment by GnstheGrain** [comment]
I've made the Spreadsheet open for anyone to edit. I'm no expert so please feel free to edit at will. 
Thanks for the tooltips idea. Will add some tonight

**Comment by mfc** [comment]
wow this is really nice suggestion! sharing with @SvenSemmler

**Comment by ben-grande** [comment]
@andrewdavidwong Is this already the case? I don't know how it was before. Is this issue asking for more fields?

**Comment by andrewdavidwong** [comment]
> [@andrewdavidwong](https://github.com/andrewdavidwong) Is this already the case? I don't know how it was before. Is this issue asking for more fields?

Yes, it's asking for more fields. You can see what the user had in mind in the example spreadsheet he provided: https://docs.google.com/spreadsheets/d/1h8Ko-NIhh9r3h52yatt85UFDzBWj3kkk2MI6zmkAQO8/
